projects/ng-components/src/lib/controls/advanced-search/models.ts

```diff
@@ -1,5 +1,9 @@
 import { formatDate } from "@angular/common";
```

Comment:
στον constructor του FilterClause βλέπω:
`    this.uiOperator = this.dataType === 'datetime' ? '' : Operators.EQUALS.label; // that's fine (for now)...`
Δεν με γεμίζει με αισιοδοξία πως γίνονται σωστά τα request προς τον server

Reply:
Αυτό θα μείνει ως έχει γιατί στην περίπτωση του datetime δεν θέλουμε να έχουμε visible operator στο UI καθώς είναι της μορφής "Από Ημ/νια" και "Εως Ημ/νια", γιαυτό και επιστρέφει κενό string. Στις περιπτώσεις eq/neq/contains πλέον θα επιστρέφει το σωστό uiOperator